Client/Configuration/Fennec.xcconfig

```diff
@@ -12,4 +12,13 @@ INCLUDE_SETTINGS_BUNDLE = YES
 
 // Adjust SDK Variables
 ADJUST_ENVIRONMENT =
-ADJUST_APP_TOKEN = 
+ADJUST_APP_TOKEN =
+
+// The suffix of the app group identifier used for sharing data across
+// extensions and keychain
+MOZ_GROUP_SUFFIX = org.mozilla.ios.Firefox
+
+// Leanplum SDK Variables
+LEANPLUM_ENVIRONMENT = development
+LEANPLUM_APP_ID = app_MUd3n2yw07Lg5hy0f8hRXuj1jI5ml17ww3haFrbKUBw
```

Comment:
We should also do the development key via BuddyBuild. And not leave it in the repo.